src/plugins/console/server/lib/spec_definitions/json/generated/ingest.delete_geoip_database.json

```diff
@@ -0,0 +1,31 @@
+{
+  "ingest.delete_geoip_database": {
+    "url_params": {
+      "error_trace": "__flag__",
+      "filter_path": [],
+      "human": "__flag__",
+      "pretty": "__flag__",
+      "master_timeout": [
+        "30s",
+        "-1",
+        "0"
+      ],
+      "timeout": [
+        "30s",
+        "-1",
+        "0"
+      ]
+    },
+    "methods": [
+      "DELETE"
+    ],
+    "patterns": [
+      "_ingest/geoip/database/{id}"
+    ],
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/TODO.html",
```

Comment:
Do we still not have docs for these API's (delete, get, put)? Wonder if we should still show the documentation link at all in that case since it points to a 404 page

Reply:
Great catch, thanks! This is actually a typo in the Elasticsearch repo, I opened https://github.com/elastic/elasticsearch/pull/113082 to fix it. To get it fixed here, the normal way is:

 * fix it in Elasticsearch
 * wait or trigger the daily sync to the Elasticsearch specification
 * then run the Kibana generate_console_definitions.js script.

That's a lot! In the interest on not blocking this pull request, I have fixed the URLs in [`5bc5c77` (#193208)](https://github.com/elastic/kibana/pull/193208/commits/5bc5c779eca40ea2cb79697ee8b65580083d1583).